src/test/run-make/libtest-thread-limit/Makefile

```diff
@@ -0,0 +1,7 @@
+-include ../../run-make-fulldeps/tools.mk
+
+# only-linux
+
+all:
+	$(RUSTC) test.rs --test
```

Comment:
This probably needs a `--target $(TARGET)`, though might also need to be made host only since we need to run the resulting binary, I'm not sure.

Reply:
> This probably needs a `--target $(TARGET)`,

The log shows that `musl-gcc` was already used, so I *guess* that's already passed somewhere... But maybe `CC` was overridden. :thinking: I'll have a try... Thanks for the pointer.

> though might also need to be made host only since we need to run the resulting binary, I'm not sure.

I'm not sure either, but I noticed that `src/test/run-make-fulldeps/libtest-json` also runs the resulting binary, and its [`Makefile`](https://github.com/rust-lang/rust/blob/ee88f46bb5e27c4d9f30326e69ff2298dcbeecb1/src/test/run-make-fulldeps/libtest-json/Makefile) does not have anything special.

---

There's also some strange `/bin/dash: 1: Cannot fork` in the log. That goes away if I remove the `ulimit`, but I don't know why it goes away (or why it happens), and the compilation still fails... :confused:
The test is fine on `i586-unknown-linux-gnu` (at least locally). I'm investigating this with `--target i686-unknown-linux-musl` now.